test/e2e/performanceprofile/functests/2_performance_update/memorymanager.go

```diff
@@ -198,18 +210,13 @@ var _ = Describe("[rfe_id: 43186][memorymanager] Memorymanager feature", Label(s
 			spec, _ := json.Marshal(initialProfile.Spec)
 			// revert only if the profile changes.
 			if !equality.Semantic.DeepEqual(currentSpec, spec) {
-				Expect(testclient.Client.Patch(context.TODO(), profile,
-					client.RawPatch(
-						types.JSONPatchType,
-						[]byte(fmt.Sprintf(`[{ "op": "replace", "path": "/spec", "value": %s }]`, spec)),
-					),
-				)).ToNot(HaveOccurred())
+				profiles.UpdateWithRetry(initialProfile)
 
-				By("Applying changes in performance profile and waiting until mcp will start updating")
-				mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdating, corev1.ConditionTrue)
+				testlog.Infof("Applying changes in performance profile and waiting until %s will start updating", resourcePool)
```

Comment:
is there a specific reason for replacing `By` with `testlog.Infof`?

Reply:
I wanted to print  a log to know the poolName that's being modified  mcp or nodepool in case of hypershift.

Reply:
We can do that with `By(fmt.Sprintf(""))`

Reply:
ginkgo makes messages printed with `By` more bold and readable than just a simple log

Reply:
fixed in the latest commit